VISION.md

```diff
@@ -0,0 +1,17 @@
+# Material UI’s Philosophy
+
+We are on a mission to provide an elegant implementation of the Material Design Specification.
+But we believe we can do more.
+
+We want Material-UI to become whatever is generally useful for application development in the spirit of the Material Design spec.
+This means Material Design with a general use UI library of components that are needed by many. This general use doesn't imply any other design methodology.
+It also means we will have components or combinations that are simply not addressed in the spec.
+
```

Comment:
I'm not totally clear on what this paragraph is alluding to. What does general-use mean in this context, and what is the "more" mentioned in the previous paragraph?